flink-runtime/src/main/java/org/apache/flink/runtime/operators/coordination/util/IncompleteFuturesTracker.java

```diff
@@ -108,4 +94,22 @@ void removeFromSet(CompletableFuture<?> future) {
             lock.unlock();
         }
     }
+
+    @VisibleForTesting
+    Collection<CompletableFuture<?>> getCurrentIncomplete() {
```

Comment:
Sorry here I made a mistake that I initially want to make this method a pure getter one without modifying the state, and it would be only used in the tests for the verification. Do you think this would be also ok that I remove `incompleteFutures.clear()` and keep the name as `getCurrentIncomplete()` ?

Reply:
If `incompleteFuture.clear()` is not needed to be called in this method, then it should be fine.